files/zh-cn/web/html/element/summary/index.md

```diff
@@ -1,44 +1,145 @@
 ---
-title: <summary>
+title: <summary>：摘要展现元素
 slug: Web/HTML/Element/summary
 ---
 
 {{HTMLSidebar}}
 
-## 概述
+[HTML](/zh-CN/docs/Web/HTML) **`<summary>` 元素** 指定了 {{HTMLElement("details")}} 元素展现盒子的内容的摘要，标题或图例。点击 `<summary>` 元素切换了亲代 `<details>` 元素开启和关闭的状态。
 
-**HTML `<summary>` 元素** 用作 一个{{HTMLElement("details")}}元素的一个内容的摘要，标题或图例。
+{{EmbedInteractiveExample("pages/tabbed/summary.html","tabbed-shorter")}}
 
-> **备注：** 如果 `<summary>` 元素在浏览器内不能被正确解析和渲染，，则会展示\<details>标签内的其他内容。
+## 属性
 
-<table class="properties">
- <tbody>
-  <tr>
-   <th scope="row">标签闭合</th>
-   <td>开始标签和结束标签都不能省略。</td>
-  </tr>
-  <tr>
-   <th scope="row">父级元素</th>
-   <td>{{ HTMLElement("summary") }}元素是{{ HTMLElement("details") }} 元素的子元素。</td>
-  </tr>
-  <tr>
-   <th scope="row">文档规范</th>
-   <td><a href="http://www.whatwg.org/specs/web-apps/current-work/multipage/interactive-elements.html#the-summary-element">HTML5, section 4.11.2</a></td>
-  </tr>
- </tbody>
-</table>
+本元素仅包含[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)。
 
-## 属性
+## 使用注意
+
+`<summary>` 元素的内容可以是任意标题内容、纯文本，或是可以在段落内使用的 HTML 代码。
+
+`<summary>` 元素*仅*可作为 `<details>` 元素的第一个子节点。当用户在 `<summary>` 元素上点击时，亲代的 `<details>` 元素切换了开启和关闭的状态，然后就会向 `<details>` 元素发送 {{domxref("HTMLDetailsElement/toggle_event", "toggle")}} 事件，令用户得知这样的状态改变何时发生。
+
+### 默认标签文字
+
+如果一个 `<details>` 元素的第一个子节点不是 `<summary>` 元素，{{Glossary("user agent", "用户代理")}}会使用一个默认字符串（通常为“详细信息”）作为展现盒子的标签内容。
```

Comment:
```suggestion
如果一个 `<details>` 元素的第一个子节点不是 `<summary>` 元素，{{Glossary("user agent", "用户代理")}}会使用一个默认字符串（通常为“详情”）作为展现盒子的标签内容。
```

On Chrome, it would be:

![image](https://github.com/mdn/translated-content/assets/15844309/09d48ec6-233c-4bf9-aefe-8174ec25a91b)

Reply:
safari and edge shows '详细信息' 🤔

Reply:
How about "详细信息/详情"

Reply:
```suggestion
如果一个 `<details>` 元素的第一个子节点不是 `<summary>` 元素，{{Glossary("user agent", "用户代理")}}会使用一个默认字符串（通常为“详细信息”或“详情”）作为展现盒子的标签内容。
```